pages/VerifiableCredentials/Overview.md

```diff
@@ -0,0 +1,75 @@
+# Verifiable Credentials Specification
+__Version pre-1.3.0__
+
+[Attribute Sets](../DSNP/AttributeSets.md) and [Attestation](../ActivityContent/Associated/Attachments.md#attestation) attachments shared via DSNP reference data documents containing Verifiable Credentials and related objects.
+For DSNP purposes, certain restrictions and extensions are applied to the base World Wide Web Consortium (W3C) specification documents noted below.
+When there are DSNP extensions, they are guaranteed to use non-colliding terms.
+
+It is anticipated that these W3C specifications will evolve, and applications interacting with Verifiable Credentials, Schemas, and DIDs via DSNP are encouraged to be as flexible as possible in their syntax support as new versions and supporting software libraries become available.
+Therefore, compatibility with the following versions and specifications should be construed as the minimal level of support required to ensure interoperability between DSNP applications.
+
+## Compatibility
+
+Current usage with DSNP relies on the following specifications:
+
+| Specification | Version/Status | Relevant JSON-LD `@context` Values |
+| --- | --- | --- | 
+ | Verifiable Credentials Data Model | [1.1 (W3C Recommendation)](https://www.w3.org/TR/vc-data-model-1.1)<br>[2.0 (W3C Candidate Recommendation Draft 23 August 2024)](https://www.w3.org/TR/2024/CRD-vc-data-model-2.0-20240823/) | `https://www.w3.org/2018/credentials/v1`<br>`https://www.w3.org/ns/credentials/v2` |
+ | Verifiable Credential Data Integrity | [1.0 (W3C Candidate Recommendation Draft 3 August 2024)](https://www.w3.org/TR/2024/CRD-vc-data-integrity-20240803/) | `https://w3id.org/security/data-integrity/v2` |
```

Comment:
When I follow this link, I get the following message on the site: For the latest version, please look at [https://www.w3.org/TR/vc-data-integrity/.](https://www.w3.org/TR/vc-data-integrity/)

Reply:
Updated